src/patches/dense_slice.cpp

```diff
@@ -128,10 +127,15 @@ DenseSlice::DenseSlice(const lsqecc::Layout &layout, const tsl::ordered_set<Patc
                     static_cast<CellBoundaries>(cell)};
         }
     }
-
+    
     size_t distillation_time_offset = 0;
     for(auto t : layout.distillation_times())
-        time_to_next_magic_state_by_distillation_region.push_back(t+distillation_time_offset++);
+        if (edpclayout) {
+            time_to_next_magic_state_by_distillation_region.push_back(t);
+        }
+        else {
+            time_to_next_magic_state_by_distillation_region.push_back(t+distillation_time_offset++);           
+        }        
```

Comment:
Would be nice to have this as a separate option. But good for a separate PR

Reply:
Added it as a separate option in the newest commit.